SingularityExecutor/src/main/resources/logrotate.conf.hbs

```diff
@@ -16,6 +16,7 @@ notifempty
 {{#if extrasFiles}}
 {{#each extrasFiles}}{{{this}}} {{/each}} {
   dateformat {{{ extrasDateformat }}}
+  olddir {{{ rotateDirectory }}}
```

Comment:
I'm not sure if this is really the right approach. I rotate all extra files into the rotateDirectory, and not only service.log